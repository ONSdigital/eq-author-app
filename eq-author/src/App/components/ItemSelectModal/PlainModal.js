import styled from "styled-components";
import Modal from "components/Modals/Modal";

const PlainModal = styled(Modal).attrs({ hasCloseButton: false })`
  .Overlay {
    background: transparent;
  }

  .Modal {
    width: 23em;
    padding: 0;
    top: 8em;
  }
`;

export default PlainModal;
