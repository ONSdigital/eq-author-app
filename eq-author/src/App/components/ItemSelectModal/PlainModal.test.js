import React from "react";
import PlainModal from "App/components/ItemSelectModal/PlainModal";
import { shallow } from "enzyme";

describe("PositionModal/PlainModal", () => {
  it("should render", () => {
    const wrapper = shallow(
      <PlainModal isOpen onClose={jest.fn()}>
        <p>
          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Reiciendis
          dignissimos, ipsum laboriosam autem ipsa corrupti dicta tempore
          provident magnam saepe consectetur molestiae illo ad temporibus, quasi
          animi rem aliquid omnis.
        </p>
      </PlainModal>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
