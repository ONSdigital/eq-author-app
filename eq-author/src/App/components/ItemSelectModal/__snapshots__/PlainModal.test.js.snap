// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PositionModal/PlainModal should render 1`] = `
.c0 .Overlay {
  background: transparent;
}

.c0 .Modal {
  width: 23em;
  padding: 0;
  top: 8em;
}

<Modal
  className="c0"
  hasCloseButton={false}
  isOpen={true}
  onClose={[MockFunction]}
>
  <p>
    Lorem ipsum dolor sit, amet consectetur adipisicing elit. Reiciendis dignissimos, ipsum laboriosam autem ipsa corrupti dicta tempore provident magnam saepe consectetur molestiae illo ad temporibus, quasi animi rem aliquid omnis.
  </p>
</Modal>
`;
